src/database/config/db_build.sql

```diff
@@ -42,4 +42,19 @@ CREATE TABLE requirement (
 	prop_id INT REFERENCES proposal(id)
 );
 
+INSERT INTO field (name) 
+values ('Developer');
+
+INSERT INTO specialization (name,field_id) 
+values ('web develpoer',1);
+
+INSERT INTO users (firstname,lastname,email,freelancer_url,photo_url,mobile_number,specalization_id,password)
+values('fatma','siam','f.siam@gmail.com','https://mm.mm.mmmmm','https://mm.mm.mmmmm','05333',1,'123123');
```

Comment:
@Fatmasiam Hi! You might want to insert the hashed (and salted) version of '123123' as the password value here, so that you can actually type '123123' in the login form and have it successfully match the password in the database.

Reply:
ok, I will edit it now @tomduggan85

Reply:
Done @tomduggan85